src/components/mainContent/Navbar.js

```diff
@@ -15,6 +15,47 @@ export default function NavBar(props) {
     setDropdownOpen(!dropdownOpen);
   };
 
+  const navSections = [
+    {
+      path: "/",
+      name: "Landing Page",
+    },
+    {
+      path: "/aboutUs",
+      name: "About Us",
+    },
+    {
+      path: "/college",
+      name: "Case Study #1",
+    },
+    {
+      path: "/facebook",
+      name: "Case Study #2",
+    },
+    {
+      path: "/facialRecognition",
+      name: "Case Study #3",
+    },
+    {
+      path: "/conclusion",
+      name: "Conclusion",
+    },
+    {
+      path: "/resources",
+      name: "Resources",
+    },
+  ];
+
+  //arrow function that maps all our links to dropdown
+  const getNavlinks = (path) =>
```

Comment:
is there a reason why we need to pass in the `path` as a parameter instead of just directly using `props.path` inside the function?